### PR TITLE
Add tickets statistics

### PR DESCRIPTION
**Pull request by nasia-sam**
Compute ticket statistics as a part of #462 

*  Add 'week' column to TicketPurchases
*  Add methods in Conference to compute the statistics
*  Add line charts in Conference#index and #show
*  Add doughnut charts in Tickets#index

### CODE REVIEW

**Comment by differentreality** [comment]
Since you are adding a lot of new visual information, could you please add some screenshots of all the new things you implemented?

**Comment by nasia-sam** [comment]
* admin/conference#index
Shows total number of sold tickets for each conference
18901761_1732335100117585_1330558867_o

* admin/conference#show
Shows number of sold tickets for each ticket category for a certain conference
18983132_1732335053450923_380326229_n

* admin/tickets#index
Two charts; one shows the percentage of each ticket category and the second one shows percentage of the monetary contribution of each tixket category
18902388_1732335020117593_205472125_n

**Comment by differentreality** [comment]
@nasia-sam can you please rebase?

File: db/migrate/20170419132148_add_week_to_ticket_purchases.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  class TmpTicketPurchase < ActiveRecord::Base
+    self.table_name = 'ticket_purchases'
+  end
+
+  def change
+    add_column :ticket_purchases, :week, :integer
+
+    TmpTicketPurchase.reset_column_information
+    TmpTicketPurchase.find_each do |purchase|
+      purchase.week = purchase.created_at.strftime('%W')
```
Comment (differentreality) [inline db/migrate/20170419132148_add_week_to_ticket_purchases.rb]: It seems we might have ticket_purchase records without a created_at field. Not sure why, but let's take that into consideration.

**Comment by differentreality** [comment]
Please rebase

**Comment by differentreality** [comment]
ping @nasia-sam @AEtherC0r3

**Comment by differentreality** [comment]
Nice work @nasia-sam :+1: 
Nice work @AEtherC0r3 :+1:

**Pull request merged**